docker/nginx/conf.d/server/server.api

```diff
@@ -137,15 +137,14 @@ location /abuse/report {
 location /hns {
     include /etc/nginx/conf.d/include/cors;
 
-    # match the request_uri and extract the hns domain and anything that is passed in the uri after it
-    # example: /hns/something/foo/bar matches:
-    # > hns_domain: something
-    # > path: /foo/bar/
-    set_by_lua_block $hns_domain { return string.match(ngx.var.uri, "/hns/([^/?]+)") }
-    set_by_lua_block $path { return string.match(ngx.var.uri, "/hns/[^/?]+(.*)") }
-
-    proxy_set_header Host $host;
-    include /etc/nginx/conf.d/include/location-hns;
+    rewrite_by_lua_block {
+        local hns_domain = string.match(ngx.var.uri, "/hns/([^/?]+)")
```

Comment:
Q: Will this also match on folders called "hns"?

Reply:
Seems to be fine in my testing, e.g.:

https://dev3.siasky.dev/CABqaqxbdeRjC5hAj1v8I8ZKDPT1uNPmarTE017m40Av_g/hns/foo.txt
https://dev3.siasky.dev/hns/note-to-self/hns/foo.txt

Reply:
good good :)